scalastyle-config.xml

```diff
@@ -101,7 +101,7 @@
    <parameter name="maxLength"><![CDATA[50]]></parameter>
   </parameters>
  </check>
- <check level="warning" class="org.scalastyle.scalariform.MethodNamesChecker" enabled="true">
+ <check level="warning" class="org.scalastyle.scalariform.MethodNamesChecker" enabled="false">
```

Comment:
is this is not recommendation for method Names ?

Reply:
It doesn't allow for method names like `+` or `++` which are both acceptable and common in scala